### PR TITLE
Add VirtualMachineExport resource

### PR DESCRIPTION
**Pull request by jpeimer**
##### Short description:
Add VirtualMachineExport resource

##### More details:
About VirtualMachineExport: https://kubevirt.io/user-guide/operations/export_api/#export-virtual-machine-volumes

### CODE REVIEW

File: ocp_resources/virtual_machine_export.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+
+class VirtualMachineExport(NamespacedResource):
+    """
+    VirtualMachineExport object.
+    """
+
+    api_group = NamespacedResource.ApiGroup.EXPORT_KUBEVIRT_IO
+
+    class SourceKind:
```
Comment (myakove) [inline ocp_resources/virtual_machine_export.py:16]: Why this is needed here `class SourceKind:`?

**Review by myakove — CHANGES_REQUESTED** [review]

File: ocp_resources/virtual_machine_export.py (hunk 1 above)
Comment (jpeimer) [reply ocp_resources/virtual_machine_export.py:16]: Because VirtualMachineExport can accept only 3 types of source objects: 
- VM
- VM Snapshot
- PVC

Comment (myakove) [reply ocp_resources/virtual_machine_export.py:16]: Where this is used? In which cases do you use it?

**Review by myakove — CHANGES_REQUESTED** [review]

**Comment by myakove** [comment]
Please fix CI

File: ocp_resources/virtual_machine_export.py (hunk 1 above)
Comment (jpeimer) [reply ocp_resources/virtual_machine_export.py:17]: When we create a VirtualMachineExport object, like here:

```
    with cluster_resource(VirtualMachineExport)(
        name=f"vmexport-from-snapshot",
        namespace="default",
        client=admin_client,
        token_secret_ref="virt-export-token",
        source_api_group=VirtualMachineSnapshot.api_group,
        source_kind=VirtualMachineExport.SourceKind.VM_SNAPSHOT,
        source_name="my-vmsnapshot",
        teardown=False,
    ) as vm_export_from_snapshot:
```

File: ocp_resources/virtual_machine_export.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        token_secret_ref=None,
+        source_api_group=None,
+        source_kind=None,
+        source_name=None,
+        timeout=TIMEOUT_4MINUTES,
+        delete_timeout=TIMEOUT_4MINUTES,
+        yaml_file=None,
+        **kwargs,
+    ):
+        if not token_secret_ref:
```
Comment (myakove) [inline ocp_resources/virtual_machine_export.py:37]: you shouldn't raise on this `token_secret_ref` if the user pass yaml_file.

**Review by myakove — CHANGES_REQUESTED** [review]

File: ocp_resources/virtual_machine_export.py (hunk 2 above)
Comment (jpeimer) [reply ocp_resources/virtual_machine_export.py]: right, changed it, thanks

**Review by myakove — APPROVED** [review]

**Review by rnetser — DISMISSED** [review]

**Comment by jpeimer** [comment]
I removed source_api_group from the check because it can be None for SourceKind PVC

**Comment by jpeimer** [comment]
!verified

**Review by myakove — APPROVED** [review]

**Pull request merged**